services/fhir.ts

```diff
@@ -216,7 +216,7 @@ export function makeReference<T extends AidboxResource>(
 
 export function isReference<T extends AidboxResource>(
     resource: T | AidboxReference<T>
-): resource is AidboxReference<T> {
+): boolean {
```

Comment:
Why did you change the type guard here?

Reply:
`_.isEmpty` returns boolean
https://lodash.com/docs/4.17.15#isEmpty

Reply:
Indeed @ufocoder, but here we need type predicate behavior.
https://www.typescriptlang.org/docs/handbook/advanced-types.html#type-guards-and-differentiating-types

The same technique is using for RemoteData type https://github.com/beda-software/aidbox-react/blob/master/libs/remoteData.ts#L51